frontend/src/components/Paper.svelte

```diff
@@ -1,28 +1,37 @@
 <script>
-  import { BreadcrumbItem, Button, TableBodyCell, TableBodyRow } from "flowbite-svelte";
+    import {BreadcrumbItem, Button, TableBodyCell, TableBodyRow} from "flowbite-svelte";
 
-  export let href;
-  export let title;
-  export let category;
-  export let slots;
-  export let show_category = false;
+    export let href;
+    export let title;
+    export let id;
+    export let category;
+    export let slots;
+    export let show_slots = false;
+    export let show_category = false;
```

Comment:
```suggestion
    export let category = null;
    export let slots = null;
```
`show_slots` and `show_category` might be redundant.
By giving properties a default value we make them optional. Now we can check against the default value if they are present.

Reminder: Do not commit this suggestion right away because this change requires other changes in this and other files as well.

Reply:
I implemented your suggestion. However, the visibility of the column title as defined in Papers.svelte is also variable, and just giving Papers.svelte the slots/category properties and assigning any value to them in order for the column title to show up is pretty ugly. If you have any suggestions to remedy this, let me know.